packages/mds-policy-service/@types/intents.ts

```diff
@@ -0,0 +1,74 @@
+import { now } from '@mds-core/mds-utils'
+import type { BaseRule, MicroMobilityStatesToEvents, PolicyDomainModel } from './models'
+
+export const INTENT_TYPES = ['no_parking'] as const
+export type INTENT_TYPE = typeof INTENT_TYPES[number]
+
+export const BASE_POLICY_DEFAULTS = {
```

Comment:
I assume these properties align with some policy type. Would be nice to include the type rather than have this constant just be some inferred type that may or may not overlap with its intended uses.

Reply:
Sure thing, type added.